docs/map_differentiated-services-scaleout.rst

```diff
@@ -0,0 +1,50 @@
+.. _differentiated-services-scaleout:
+
+Differentiated Services and Scale Out
+-------------------------------------
+.. figure:: media/driver_multiple_environments.png
```

Comment:
needs to be updated to ``media/f5-lbaas-differentiated-service-environments`` and caption needs to be updated to LBaaSv2